addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -177,157 +177,231 @@ QUnit.test('add emoji replaces (keyboard) text selection', async function (asser
     await nextAnimationFrame();
 });
 
-// Test skipped until mentions manager is ready
-QUnit.skip('display partner mention suggestions on typing "@"', async function (assert) {
+QUnit.test('display partner mention suggestions on typing "@"', async function (assert) {
     assert.expect(2);
 
-    await this.start();
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'get_mention_suggestions') {
+                return [
+                    [{
+                        email: "testpartnert@odoo.com",
+                        id: 11,
+                        name: "TestPartner",
+                    }],
+                    [],
+                ];
+            }
```

Comment:
Ideally should use `this.data` and rely on mock server, but okay with that right now.